variants/web/config/environments/test.rb

```diff
@@ -0,0 +1,17 @@
+insert_into_file 'config/environments/test.rb', before: %r{Rails.application.configure do} do
+  <<~EOT
+      require_relative '../../spec/support/disable_animation'
+
+  EOT
+end
+
+insert_into_file 'config/environments/test.rb', before: %r{^end} do
+  <<-EOT
+
+  # Disable all animation during tests
+  config.middleware.use Rack::NoAnimations
+
+  # Do not fallback to assets pipeline if a precompiled asset is missed.
```

Comment:
Typo: is miss**ing**.

Reply:
Fixed 👍 - https://github.com/nimblehq/rails-templates/pull/156/commits/5c4211e206043e00d468c3377e9a9aa401a4738b